src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,92 @@
+import { useParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+import { Loader } from './Loader/Loader';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
you can initially set loading to true, and then make it falsy, when your data is fecthed